scripts/removeOldKibanaIndices.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+logFile = open('/var/log/probe/KibanaStartup.log', 'w')
+
+res = requests.get('http://localhost:9200/_cat/indices/.kibana_*?s=index')
+content = res.content
+kibanaIndexNums = []
```

Comment:
Can we change this to just kibanaIndices now

Reply:
Yep! Sorry, I'd made that change, but haven't pushed it yet. Re-testing to make sure I changed it everywhere correctly.